rust/src/wallet.rs

```diff
@@ -55,6 +55,7 @@ pub fn maker_peer_info() -> PeerInfo {
     }
 }
 
+#[derive(Clone, derive_more::Display)]
```

Comment:
Not necessary to action, but for the future: I don't think we should add an extra library for this but just implement it ourselves.